examples/powershell/vro/stack.yml

```diff
@@ -0,0 +1,16 @@
+provider:
+  name: openfaas
+  gateway: https://veba.primp-industries.com
+functions:
+  powershell-vro:
+    lang: powercli
+    handler: ./handler
+    image: vmware/veba-powershell-vro:latest
+    environment:
+      write_debug: true
+      read_debug: true
+      function_debug: false
+    secrets:
+      - vro-secrets
+    annotations:
+      topic: DrsVmPoweredOnEvent
```

Comment:
Probably does not harm to set to `"DrsVmPoweredOnEvent,VmPoweredOnEvent"` so non-DRS users have a good experience.

Reply:
Yup, I wonder if we need to be consistent with the other examples too

Reply:
Absolutely, will file an issue

Reply:
👍